src/shared/default-routes.js

```diff
@@ -17,7 +17,7 @@ export default ({
     allowEmptyResponse: true
   }
 }, {
-  path: '/:year/:monthnum/:day/:postname',
+  path: '/:category/:year/:monthnum/:postname',
```

Comment:
Ah I didn't realise it was the Wordpress default, I'll revert it. It'd be nice to rely on the default Tapestry routes, lets chat more about it tomorrow, I think this task could do with some short term/long term definition.